iosApp/PhongKMMIC/Sources/Presentation/Modules/LogIn/LogInCombineViewModel.swift

```diff
@@ -14,11 +14,18 @@ import shared
 
 final class LogInCombineViewModel: ObservableObject {
 
-    @Published var email = ""
-    @Published var password = ""
+    #if DEBUG
```

Comment:
Can we add something like `#if RUNNING_TEST`? 
Using `DEBUG` here can lead to incorrect behavior when running the app in a non-release build configuration 🤔

Reply:
Oh, I want it to prefill the text fields for me so I don't have to do that :D. Each time the test runs the keychain will be clear so I have to re-login. Because it is in the native level and didn't touch any shared part so it won't affect the app flow.